crates/red_knot_module_resolver/src/resolver.rs

```diff
@@ -120,6 +124,9 @@ pub(crate) fn file_to_module(db: &dyn Db, file: VfsFile) -> Option<Module> {
 /// Configures the search paths that are used to resolve modules.
 #[derive(Eq, PartialEq, Debug)]
 pub struct ModuleResolutionSettings {
+    /// The target Python version the user has specified
+    pub target_version: SupportedPyVersion,
```

Comment:
Nit: I'm inclined to rename `SupportedPyVersion` to `TargetPyVersion` considering that you named the field very consistently `target_version`

Reply:
I see the logic there, but I already have a struct called `TargetPyVersion` in `supported_py_version.rs`. What do I call that if I give that name to the enum currently called `SupportedPyVersion`?